src/utils/GAUtils.js

```diff
@@ -11,9 +11,16 @@ export const getUserPseudoId = ga_measurement_id => {
   return new Promise((resolve, reject) => {
     try {
       if (typeof window !== 'undefined' && typeof window.gtag !== 'undefined') {
-        window.gtag('get', ga_measurement_id, 'client_id', field =>
-          resolve(field),
-        );
+        window.gtag('get', ga_measurement_id, 'client_id', resolve);
+        /* 
+          In Brave browser, window.gtag execution will be terminated without 
+          throwing error, so this promise will not never being resolved or rejected.
```

Comment:
It looks like window.gtag will call callback (`resolve`) in infinite time, so we add timeout here?

Suggest from chatgpt, remember to add `clearTimeout` to clear the timer memory

```js
const t = setTimeout(() => {
          console.error('Unable to resolve GA User Pseudo Id');
          resolve(null);
        }, 500);

window.gtag('get', ga_measurement_id, 'client_id', field => { clearTimeout(t); resolve(field); });
```

Reply:
Nice suggestion, updated and tested
- [x] use chrome browser, submit interview / salary  successfully, check the API request body contains ga_user_pseudo_id
- [x] use brave browser, submit interview / salary  successfully